### PR TITLE
Make both Cabins fully type-safe.

### LINKED ISSUE #180: Deep Static Analysis and Unit Testing
We currently have [Psalm](https://github.com/vimeo/psalm) running in Travis CI on the core files (src/Engine). We should expand the coverage include everything in `src`. We'll have to do each Cabin individually.

### PR DESCRIPTION
**Pull request by paragonie-scott**
Closes #180

### CODE REVIEW

File: src/Cabin/Bridge/Model/UserAccounts.php [hunk 1]
```diff
@@ -39,6 +39,9 @@ class UserAccounts extends ModelGear
         (self::RECOVERY_TOKEN_BYTES * 4 / 3)
     );
 
+    /** @var Zxcvbn $zxcvbn */
```
Comment (muglug) [inline src/Cabin/Bridge/Model/UserAccounts.php]: U wot mate?

Comment (paragonie-scott) [reply src/Cabin/Bridge/Model/UserAccounts.php]: It's an actual class name :P https://github.com/bjeavons/zxcvbn-php

File: src/Cabin/Bridge/psalm.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+>
+    <projectFiles>
+        <directory name="src/Cabin/Bridge" />
+    </projectFiles>
+    <stubs>
+        <file name="vendor/paragonie/halite/stub/Sodium.stub.php" />
+    </stubs>
+    <issueHandlers>
+        <PropertyNotSetInConstructor errorLevel="suppress" />
+        <TooFewArguments errorLevel="suppress" />
```
Comment (muglug) [inline src/Cabin/Bridge/psalm.xml]: is this due to `func_get_args` usage?

**Comment by muglug** [comment]
Congrats! You didn't happen to use [Psalter](https://github.com/vimeo/psalm/releases/tag/0.3.73) to add missing types, did you?

File: src/Cabin/Bridge/psalm.xml (hunk 2 above)
Comment (paragonie-scott) [reply src/Cabin/Bridge/psalm.xml]: Copypasta.

**Comment by paragonie-scott** [comment]
No, I saw that on Reddit but forgot it existed.

**Pull request merged**